pkg/crd/k8c.io/kubermatic.k8c.io_clusters.yaml

```diff
@@ -985,7 +990,6 @@ spec:
                       required:
                         - infraManagementUser
                         - storagePolicy
-                        - vmNetName
```

Comment:
We completely lost this validation. We should do this via code instead.

Reply:
This validation is wrong, `vmNetName` or `networks`(the new field) are not mandatory since if someone has attached network(s) to their VM template they don't need to provide this at the cluster or machine deployment level.